blockchain/v1/reactor_test.go

```diff
@@ -116,15 +116,8 @@ func newBlockchainReactor(
 	sm.SaveState(db, state)
 
 	// let's add some blocks in
+	lastCommit := types.NewCommit(0, 1, types.BlockID{}, nil)
```

Comment:
Perhaps I'm missing something, but shouldn't be the `lastCommit` initialised for `blockHeight-1` rather than `0`. It is logical based on the context.